SAMtags.tex

```diff
@@ -453,6 +453,165 @@ \subsubsection{Color space}
 Color read quality on the original strand of the read. Same encoding as {\sf QUAL}; same length as {\tt CS}.
 \end{description}
 
+\section{Draft tags}
+
+These are tags which have been proposed and are broadly accepted to
+become standard tags, but a review or probationary period has been
+deemed useful.  They use the locally-defined tag namespace and
+processing software should consider that the tags may have local usage
+for other purposes.
+
+\begin{center}\small
+% This table is sorted alphabetically
+\begin{longtable}{ccp{12.5cm}}
+  \hline
+  {\bf Tag} & {\bf Type} & {\bf Description} \\
+  \hline
+  \endhead
+  {\tt Mm} & i & Base modifications / methylation  \\
+% {\tt MP} & i & Base modification qualities \\
+  {\tt Ml} & i & Base modification probabilities \\
+\end{longtable}
+\end{center}
+
+\subsection{Base modifications}
+
+Base modifications, including base methylation, are represented as a series of edits from the primary unmodified sequence as originally reported by the sequencing instrument.
+This potentially differs to the sequence stored in the main SAM {\sf SEQ} field if the latter has been reverse complemented, in which case SAM {\sf FLAG} 0x10 must be set.
+This means modification positions are also recorded against the original orientation (i.e. starting at the 5' end), and count the original base types.
+
+Each modified base listed also has a quality value associated with it.
```

Comment:
since we are not sure that phred is appropriate for the values we will be seeing, the part that discusses quality should be commented out here too, right?

Reply:
Hmm interesting semantics here.  Quality value to me doesn't imply Phred.  It simply implies a numeric value indicating the quality, which may or may not be calibrated (let's face it many often aren't!) and may or may not imply phred scale.

However we should look for a better term.  Maybe "likelihood" instead of "quality value"?  That tends to mean something very specific to some stats people.  Maybe "probability associated with it" instead then?